machine_configuration/README.md

```diff
@@ -87,6 +87,18 @@ $ lsmod | grep ast
 ```
 Ensure that ast driver is not loaded.
 
+# Machine Configuration for Provisioning Intel QAT
+
+* Turn on `intel_iommu` kernel parameter for QAT and load `vfio-pci` at boot
```

Comment:
Can we change it to
 Turn on `intel_iommu` kernel parameter and load `vfio-pci` driver at boot for QAT provisioning